### ISSUE TITLE
Integration with RNN

### ISSUE THREAD

**Issue by Startonix** [Startonix/ModularAI-#82]
import nengo
import numpy as np
import tensorflow as tf

def create_neuromorphic_network(input_signal, dimensions=1, neurons=100):
    model = nengo.Network()
    with model:
        input_node = nengo.Node(output=input_signal)
        ens = nengo.Ensemble(neurons, dimensions)
        nengo.Connection(input_node, ens)
        output_probe = nengo.Probe(ens, synapse=0.01)
    return model, output_probe

def run_neuromorphic_simulation(model, duration=1.0):
    with nengo.Simulator(model) as sim:
        sim.run(duration)
    return sim

def create_rnn_model(input_shape):
    model = tf.keras.Sequential([
        tf.keras.layers.SimpleRNN(50, activation='relu', input_shape=input_shape),
        tf.keras.layers.Dense(10, activation='softmax')
    ])
    model.compile(optimizer='adam', loss='sparse_categorical_crossentropy', metrics=['accuracy'])
    return model

def neuromorphic_input_signal(t):
    return np.sin(2 * np.pi * t)

# Create and run neuromorphic network
neuromorphic_model, neuromorphic_probe = create_neuromorphic_network(neuromorphic_input_signal)
neuromorphic_sim = run_neuromorphic_simulation(neuromorphic_model, duration=1.0)
neuromorphic_output = neuromorphic_sim.data[neuromorphic_probe]
rnn_input = neuromorphic_output.reshape((neuromorphic_output.shape[0], 1, neuromorphic_output.shape[1]))

# Create and train RNN model
rnn_model = create_rnn_model((1, neuromorphic_output.shape[1]))
rnn_model.fit(rnn_input, np.random.randint(10, size=(neuromorphic_output.shape[0],)), epochs=5)
